pkg/webui/console/store/selectors/applications.js

```diff
@@ -75,6 +77,15 @@ export const selectApplicationsTotalCount = state => selectAppsTotalCount(state)
 export const selectApplicationsFetching = state => selectAppsFetching(state)
 export const selectApplicationsError = state => selectAppsError(state)
 
+export const isOtherClusterApp = app => {
+  const isOtherCluster =
+    getHostnameFromUrl(selectAsConfig().base_url) !== app.application_server_address &&
+    getHostnameFromUrl(selectNsConfig().base_url) !== app.network_server_address &&
+    getHostnameFromUrl(selectJsConfig().base_url) !== app.join_server_address
+
+  return isOtherCluster
+}
+
```

Comment:
This isn't a selector, so it's better to have this moved into its own module `pkg/webui/console/lib/application-utils.js`.